Short-Answer/Algorithms_Answers.md

```diff
@@ -1,4 +1,25 @@
-1.  well the first code snippet works out to n^3 but i dont recognize that so
+snippet 1:  well the first code snippet works out to n^3 but i dont recognize that so
 maybe it's O(n^2).
 
-2.
\ No newline at end of file
+snippet 2:  O(n^2) this one 
```

Comment:
the first three loops are nested and are going to run n number of times. For the fourth loop ```for l in range(k + 1, 10 + k)```, k doesn't increase in this loop, so it's going to just run in constant time. Overall, this snippet is O(n^3)